### ISSUE TITLE
Quit game to RetroArch menu instead of closing RetroArch

### ISSUE THREAD

**Issue by InterClaw** [schellingb/dosbox-pure#431]
Any chance of getting an option that drops you on the RetroArch menu when you quit a game (using the game's own funtionality for this), instead of closing down RetroArch completely? I.e. so that it behaves like if you pick "Close content" on the quick menu?

**Comment by schellingb** [comment]
What system do you get this behavior?

**Comment by InterClaw** [comment]
When I read your question, that got me thinking. I just assumed this was intended behavior.

So what I did was I set up a separate RetroArch folder from scratch (v1.16.0.3, Windows 10), downloaded just DOSBox Pure and put one game in there. Set it to auto start on the menu.

Then quit and waited the 5 seconds.

... and was back in the RetroArch menu!

So it must be some weird setting I have that's doing this, so I need to figure out what. :)

**Comment by InterClaw** [comment]
... and after some looking around I found that I had:

`load_dummy_on_core_shutdown = "false"`

This needs to be "true", which is also the default it seems.

**Comment by InterClaw** [comment]
And here it is in the UI, under Settings > Core. Pretty obvious now. 😊 

![image](https://github.com/schellingb/dosbox-pure/assets/48981208/cea50089-bc7a-4f1e-ae87-5d81d10cf280)

**Comment by schellingb** [comment]
Oh, glad you figured it out!
A related question, do you like that behavior?
I never really did and I was about to remove the automatic shutting down of the core altogether. Console emulator cores don't do anything like that and since shutting down with the Quick Menu is so easy, it seems a bit unnecessary. But maybe it can continue to work like it does now, it already is an option (Emulation -> Advanced > Start Menu).

**Comment by InterClaw** [comment]
Not entirely sure what you meant now, so I'll answer in two different ways and hope one of them is the one you're looking for. :)

1.
Did you mean the RetroArch option above now? Are you behind this even existing in RetroArch? I guess DOSBox Pure is one of the more common cores that would be able to leverage this functionality, what with being able to "quit to DOS", but perhaps not the only one. Yeah, consoles don't have this. You just "close content" via the quick menu. For me, being able to close out of RA completely (with the above option turned off) was definitely not the behavior I was looking for. But it's default kept on, so I think the option might as well remain. There's always someone who likes it I guess. I probably turned that off in RA long before I had even started using DOSBox Pure, not realizing what it was. 😄 

2.
The 5 second wait to hit a key to get back to the DOS start menu is honestly not something I really find the need for. I should probably just change that setting to "Show at start, shut down core immediately after auto started game exit". It's not that often I would find the need to run another executable after having played a game. If I needed to get to the DOS start menu again, I can just "Run" the content again from the RA menu. And if I have autostart set, then just use the L2/R2 + "Restart" override.


In summary, the behavior I would be most comfortable with would be that quitting to DOS inside a game, would behave exactly the same as "Close content" on the quick menu. So I'll probably set it up like that now. 👍
